pod-operation/src/state_machine.rs

```diff
@@ -156,34 +156,31 @@ impl StateMachine {
 		// self.hvs.disable()
 	}
 
-	fn handle_init(Data(_data): Data<String>, ack: AckSender) {
+	fn handle_init(ack: AckSender) {
 		info!("Received init from client");
-		//socket.emit("init", "init").ok();
 		ack.send("init").ok();
 		Self::modify_state(State::Init);
 	}
```

Comment:
Chore: the FSM would initialize itself rather than being told to do so, so we should be able to remove this handler, unless we have reason to add a new action to reset the pod back to a parked state.